Monitoring in a Box/Ansible/group_vars/all.yml

```diff
@@ -60,4 +31,12 @@ DNSIP: 10.0.1.53
 DNSdomain: example.com
 
 ## SSH Jumphost
-SshJumpHost: ProxyJump [key in the public ip and port number]
+SshJumpHost: ProxyJump 111.223.106.169:22222
+
+DBpass: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          32393339373438313935656365646337316466386635316164323533656138323230343862373032
+          3062373738363362393532626262343237333137346534350a616461366665653462383630313665
+          33613936663864343834643838376365356633643362623534326264306537373165623539616363
+          6633393132623537380a363863333333323761373232396438323530666133396366383538643436
+          3463
```

Comment:
The database password should be a hostvar. Each deployment should have its own password.

Reply:
I was thinking of putting it into host_var at first, but on the WebGui side, the password is hard-coded into the python scripts. Therefore it would not be change, thats why i decided to put at group_vars

Reply:
Don't hard-code the password into the python script. What needs to be done is to put this script into the role's `template` folder and then use Ansible's `template` module to paste in the password.

Reply:
I will discuss with john to edit it accordingly

Reply:
Resolved in `Remove hardcoded DB password` commit.